test/dlc_tests/conftest.py

```diff
@@ -626,6 +627,16 @@ def pt15_and_above_only():
 def pt14_and_above_only():
     pass
 
+@pytest.fixture(scope="session")
```

Comment:
nit: Say we have multiple ranges to skip (a-->b and y-->z) do we expect the test writer to make multiple calls to this function, one for each range?

```
outside_versions_skip(<img_uri>, a, b)
outside_versions_skip(<img_uri>, y, z)
```

We can avoid this by using a list of tuples `[(lower, upper),]` as a param instead.

Reply:
got it, but I think we can develop this feature when it's needed later? We have not encounted an scenario like this?